specifications/factoring/factoring.yml

```diff
@@ -1447,6 +1447,80 @@ components:
                     type: string
                     description: The type of the fee
                     example: reminder
+            feeSettings:
+              type: array
+              items:
+                type: object
+                required:
+                  - rateType
+                  - feeReferenceAmount
+                  - feeTrigger
+                  - billingMethod
+                  - percentageValue
+                properties:
+                  rateType:
+                    type: string
+                    description: Specify the method used to calculate the fee amount
+                    enum:
+                      - flat
+                      - variable
+                    example: Flat
```

Comment:
The example here should be one of the enums (thus lowercase)

Reply:
Great catch